packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -253,12 +253,11 @@ export function createModelForService(
     const inputClient = {
       Name: getClientName(client),
       Description: clientDesc,
-      Operations: [],
+      Operations: [] as InputOperation[],
```

Comment:
i think this is unnecessary.
We should do this if we want the type to be explicit:
```typescript
const inputClient: InputClient = {};
```
and the `Operations` in this case should automatically be `InputOperation[]`